content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/quickstart-for-configuring-your-minio-storage-bucket-for-github-actions.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Quickstart for configuring your MinIO storage bucket for GitHub Actions
```

Comment:
Let's rename this to just "Configuring your MinIO storage bucket for GitHub Actions."